src/Standards/Squiz/Docs/WhiteSpace/MemberVarSpacingStandard.xml

```diff
@@ -0,0 +1,88 @@
+<documentation title="Member Var Spacing">
+    <standard>
+    <![CDATA[
+    There should be one blank line before the first property (member variable).
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: One blank line before the first property.">
+        <![CDATA[
+class MyClass
+{<em>
+
+</em>    protected $var1 = 'value';
+}
+        ]]>
+        </code>
+        <code title="Invalid: Incorrect number of blank lines before the first property.">
+        <![CDATA[
+class MyClass
+{<em>
+</em>    protected $var1 = 'value';
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There should be one blank line between properties.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: One blank line between each property.">
+        <![CDATA[
+trait MyTrait {
+
+    public $var1 = 'value';<em>
+
+</em>    public $var2 = 'value2';
+}
+        ]]>
+        </code>
+        <code title="Invalid: Incorrect number of blank lines between each property.">
+        <![CDATA[
+trait MyTrait {
+
+    public $var1 = 'value';<em>
+
+
+</em>    public $var2 = 'value2';
```

Comment:
Would this work ?

```suggestion
trait MyTrait {

    public $var1 = 'value';<em>


</em>    public $var2 = 'value2';<em>
</em>    public $var3 = 'value3';
```

(to show both too much and too little space)

Reply:
Updated in 72dfff061